pkg/rtc/participant.go

```diff
@@ -1697,6 +1697,12 @@ func (p *ParticipantImpl) onDataMessage(kind livekit.DataPacket_Kind, data []byt
 		//    and pushing it to all subscribers on some cadence and subscribers have their own cadence of
 		//    processing/batching and sending to edge clients.
 		p.metricTimestamper.Process(payload.Metrics)
+	case *livekit.DataPacket_RpcRequest:
+		shouldForwardData = true
+	case *livekit.DataPacket_RpcResponse:
+		shouldForwardData = true
+	case *livekit.DataPacket_RpcAck:
+		shouldForwardData = true
```

Comment:
This is enough. But, looks like this is not on the latest code. Can you please update? There are some `nil` checks. Also, don't need to set this flag. It defaults to `true` above. So, this can be something like
```
        case *livekit.DataPacket_RpcRequest:
		if payload.RpcRequest == nil {
                     return
                 }
	case *livekit.DataPacket_RpcResponse:
		if payload.RpcResponse == nil {
                     return
                 }
	case *livekit.DataPacket_RpcAck:
		if payload.RpcAck == nil {
                     return
                 }
```

Also, noticed that `ChatMessage` ☝️ is setting `shouldForwardData = true`. I think that can also be removed as the default is `true`.

Reply:
done - thank you!